Makefile

```diff
@@ -41,42 +43,72 @@ lint:
 	docker run --rm -v $(CURDIR):/dss -w /dss golangci/golangci-lint:v1.26.0 golangci-lint run --timeout 5m -v --disable-all  -E staticcheck --skip-dirs '^cmds/http-gateway,^pkg/logging'
 
 pkg/api/v1/ridpb/rid.pb.go: pkg/api/v1/ridpb/rid.proto
-	protoc -I/usr/local/include -I.   -I$(GOPATH)/src   -I$(GOPATH)/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.14.3/third_party/googleapis   --go_out=plugins=grpc:. $<
+	docker run -v$(CURDIR):/src:delegated -w /src $(GENERATOR_TAG) protoc \
```

Comment:
I think it would probably be good to specify an explicit image version for protoc in case there's a breaking change (same thing in general for other docker runs that don't involve images we built)

Reply:
The image tag already is `$(GENERATOR_TAG)`, just invoking the `protoc` executable in that image (that we have built).